hibernate-core/src/main/java/org/hibernate/internal/NaturalIdMultiLoadAccessStandard.java

```diff
@@ -34,7 +34,7 @@ public class NaturalIdMultiLoadAccessStandard<T> implements NaturalIdMultiLoadAc
 
 	private Integer batchSize;
 	private boolean returnOfDeletedEntitiesEnabled;
-	private boolean orderedReturnEnabled = true;
+	private boolean orderedReturnEnabled = false;
```

Comment:
Why was this default changed? The jdoc of `NaturalIdMultiLoadAccess#enableOrderedReturn` says:

```
By default, the returned list is ordered and the positions of the
entities correspond to the positions of their ids. [...]
```

This seems to break that assumption, no?